File: MedicalConsultation.Consultation.Api/Application/Models/ConsultationModel.cs
using System.Text.Json.Serialization;

namespace Application.Models
{
    public class ConsultationModel
    {
        [JsonPropertyName("id")]
        public string Id { get; set; } = string.Empty;

        [JsonPropertyName("patientDocument")]
        public required string PatientDocument { get; set; } = string.Empty;

        [JsonPropertyName("doctorId")]
        public required string DoctorId { get; set; } = string.Empty;

        [JsonPropertyName("consultationDate")]
        public required DateTime ConsultationDate { get; set; }
    }
}

File: MedicalConsultation.NearbyConsultations.Cron/internal/domain/entities/consultation_model.go
package entities

import "time"

type ConsultationModel struct {
	Id               string    `json:"id"`
	PatientDocument  string    `json:"patientId"`
	ConsultationDate time.Time `json:"consultationDate"`
	DoctorId         string    `json:"doctorId"`
	Specialit        string    `json:"speciality"`
}
